languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
Ah, yes, I see. But at least in that case the message still makes sense "This discussion was originally visible to a limited audience". So I AM being careful about who I share it with KTHXBAI!